src/lib.rs

```diff
@@ -168,10 +168,14 @@ impl Client {
         I: IntoIterator<Item = K>,
         K: AsRef<[u8]>,
     {
-        self.conn.write_all(b"get ").await?;
+        self.conn.write_all(b"get").await?;
         for key in keys {
-            self.conn.write_all(key.as_ref()).await?;
+            const MAX_KEY_LENGTH: usize = 250;
+            if key.as_ref().len() > MAX_KEY_LENGTH {
+                continue;
+            }
             self.conn.write_all(b" ").await?;
+            self.conn.write_all(key.as_ref()).await?;
```

Comment:
Fixed this to remove the trailing whitespace at the end of the command to compensate for [this rollback](https://github.com/Shopify/async-memcached/pull/61).